internal/query/query.go

```diff
@@ -44,7 +45,7 @@ func Async(fn func(host string) interface{}, hosts []string) map[string]any {
 
 	v := viper.GetViper()
 	if !v.GetBool("json") {
```

Comment:
I think we could do one better, and remove this conditional throughout both `query.go` and `set.go`. The conditional's intention was to ensure `--json` didn't output any non-JSON, but if this output is always directed to `os.Stderr` then it doesn't matter.

Reply:
I'll tackle this in #50